lxd/daemon.go

```diff
@@ -1347,6 +1348,14 @@ func (d *Daemon) init() error {
 		d.oidcVerifier = oidc.NewVerifier(oidcIssuer, oidcClientID, oidcAudience)
 	}
 
+	// Setup OpenFGA authorization.
+	if openfgaAPIURL != "" && openfgaStoreID != "" {
+		d.openFGAClient, err = openfga.NewClient(openfgaAPIURL, openfgaAPIToken, openfgaStoreID)
+		if err != nil {
+			return err
```

Comment:
Does this introduce a similar issue to what we recently fixed in https://github.com/canonical/lxd/pull/11917 ?